dev-server/documents/html/vitalsource-epub.mustache

```diff
@@ -70,7 +70,7 @@
           const styles = document.createElement('style');
           styles.innerHTML = `
             iframe {
-              width: 600px;
+              width: 100%;
```

Comment:
This makes the VS EPUB demo better match the real VS viewer, and it also makes this iframe trigger heuristics which `Guest` uses to determine whether an annotatable iframe should respond to side-by-side mode being activated.